rfcs/react-components/convergence/long-term-support-plan.md

```diff
@@ -0,0 +1,122 @@
+
+## Background
+
+We currently support v7, v8, and v9 versions of Fluent UI React and v0 of Northstar/Stardust.
+
+- v0 is active in the master branch. It is still supported by Teams, but all new feature work happens in v9.
+- v7 is in sunset. Only critical issues are fixed. The last fix was over 6 months ago (2023-08-17).
+- v8 is active in the master branch. Important issues raised by partners are fixed and partners actively develop on v8.
+  = v9 is active in the master branch. It is Fluent's focus for feature development.
+
+## Problem statement
+
+We need to:
+
+1. Communicate LTS plans for each version to partners and the open-source community.
+2. Plan for sunset/retirement of old versions.
+3. Coordinate with partners that contribute to or own packages in our repository.
+
+## Proposal: Manage versions with branches
+
+These are somewhat sequential, but work can be done as soon as it makes sense.
+
+#### Get v9 to parity with v8
+
+To make v9 the official current branch, developers need to be able to do everything they could with v8.
+
+Our status as of 2014-02:
+
+- In preview: Rating, SearchBox, TeachingPopover, Motion
+- In compat: Calendar, DatePicker, TimePicker
+- In progress Nav, SwatchPicker, PeoplePicker, Carousel,
+- Not started: Calendar, ColorPickerCompat,
+- On hold: Coachmark
+- Unknown: Chart, Keytips, MarqueeSelection, ActivityItem
+
+Our planning cadence is scheduling component work to get to parity.
+
+Components specific to one product should be moved into the partner repository. Components with limited shared re-use should be moved into the community contrib repository. Lower-value components or eccentric features should be considered for deprecation.
+
+We should set a date when we expect to be at parity and communicate it to partners.
+
+#### Publish an LTS policy with each version
+
+We should add a Long Term Support section to the README.MD of each version. It should detail what stage the version is in, what kind of issues will be fixed by our team, what kind of issues should be fixed by the community, and the expected response time to issue triage and PR review.
+
+We should add pointers from the wiki FAQ back to the README.MD for each version.
+
+#### Retire v7
+
+We should announce an end-of-life to v7. Proposed: June 30th, 2024. We would lock the branch from any changes, disable and archive any build pipelines, and replace public documentation sites with a redirect to a notice that v7 is retired.
+
+Any partners wanting to continue using v7, should take a snapshot of the code and integrate it into their own repositories and build it.
+
+#### Separate v0, v8 and v9 branches
+
+v7 is already in its own branch. v0, v8, and v9 are in the master branch which causes some confusion for developers.
+
+1. Create a v8 branch from master
+2. Create a northstar-v0 branch from master
+3. Remove all v9 code from the northstar-v0, and v8 branches
```

Comment:
We should also have a point about enabling branch protection policies to the v0 and v8 branches.

Reply:
Added some numbered notes.